src/frontend/planner_test/tests/testdata/output/over_window_function.yaml

```diff
@@ -44,15 +101,47 @@
     Caused by:
       Feature is not yet implemented: non-const `offset` of `lead` function is not supported yet
     No tracking issue yet. Feel free to submit a feature request at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Ffeature&template=feature_request.yml
-- sql: |
-    create table t(x int);
-    select 1+row_number() over(ORDER BY x) from t;
-  planner_error: |-
-    Feature is not yet implemented: window function in expression: (1:Int32 + row_number() OVER(ORDER BY $0 ASC))
-    No tracking issue yet. Feel free to submit a feature request at https://github.com/risingwavelabs/risingwave/issues/new?labels=type%2Ffeature&template=feature_request.yml
-- sql: |
+- id: lag with over clause
+  sql: |
+    create table t(x int, y int);
+    select x, y, lag(x) over(PARTITION BY y ORDER BY x) from t;
+  logical_plan: |-
+    LogicalProject { exprs: [t.x, t.y, first_value] }
+    └─LogicalOverWindow { window_functions: [first_value(t.x) OVER(PARTITION BY t.y ORDER BY t.x ASC ROWS BETWEEN 1 PRECEDING AND 1 PRECEDING)] }
+      └─LogicalProject { exprs: [t.x, t.y, t._row_id] }
+        └─LogicalScan { table: t, columns: [t.x, t.y, t._row_id] }
+  stream_plan: |-
+    StreamMaterialize { columns: [x, y, t._row_id(hidden), lag], stream_key: [t._row_id], pk_columns: [t._row_id], pk_conflict: NoCheck }
+    └─StreamExchange { dist: HashShard(t._row_id) }
```

Comment:
Should we have the stream key also include the partition key `y` here?

Reply:
Yes, it's fixed